src/OpenLoco/OpenLoco.cpp

```diff
@@ -125,39 +125,54 @@ namespace OpenLoco
         return _screen_age;
     }
 
-    uint8_t getScreenFlags()
+    uint16_t getScreenFlags()
     {
-        return _screen_flags;
+        return _screenFlags;
+    }
+
+    void setScreenFlags(uint16_t newScreenFlags)
```

Comment:
Is there any better name we can come up with apart from setScreenFlags and setScreenFlag for the or and = versions?

Reply:
Renamed to setAllScreenFlags